### PR TITLE
Vault stores s3 secrets

### PR DESCRIPTION
**Pull request by daisieh**
Pytest works. These changes will be tested in a corresponding CanDIGv2 pull request: https://github.com/CanDIG/CanDIGv2/pull/142.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -38,8 +38,10 @@ ARG opa_url
 ARG candig_auth
 ARG minio_url
 ARG minio_bucket_name
+ARG vault_url
 RUN sed -i s@\<CANDIG_OPA_SECRET\>@${opa_secret}@ config.ini \
```
Comment (shaikh-rashid) [inline Dockerfile]: Adding secrets in at build is not recommended. If this image is published to repo, the secret is saved inside the image.

File: entrypoint.sh [hunk 2]
```diff
@@ -2,17 +2,27 @@
...
-	rm initial_setup
+    if [[ -f "/run/secrets/cert.pem" ]]; then
+        cat /run/secrets/cert.pem >> /usr/local/lib/python3.7/site-packages/certifi/cacert.pem
+    fi
+    
+    ACCESS=$(cat /run/secrets/access)
+    sed -i s@\<MINIO_ACCESS_KEY\>@$ACCESS@ config.ini
+    
+    SECRET=$(cat /run/secrets/secret)
+    sed -i s@\<MINIO_SECRET_KEY\>@$SECRET@ config.ini
```
Comment (shaikh-rashid) [inline entrypoint.sh]: move the `sed` actions from `Dockerfile` into `entrypoint.sh`

File: Dockerfile (hunk 1 above)
Comment (daisieh) [reply Dockerfile]: I fixed this in a later commit.

File: entrypoint.sh (hunk 2 above)
Comment (daisieh) [reply entrypoint.sh]: done

**Comment by daisieh** [comment]
Sorry about the confusion; I actually forgot to take out a bunch of previously hardcoded variables for Minio that we don't use anymore.

**Pull request merged**